db/migrate/20230712135721_create_likes.rb

```diff
@@ -7,4 +7,4 @@ def change
       t.timestamps
     end
   end
-end
+end
```

Comment:
- [OPTIONAL] In this [schema](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the `User` has many `Likes` through the column `author_id`. You need to mirror this relationship by properly configuring the migration file that creates the Likes model. I would advise you to make your migration file look this way 😎 👍:

```ruby
create_table :likes do |t|
    t.timestamps
    t.references :author, foreign_key: { to_table: 'users' }, null: false, index: true 
    t.references :post, foreign_key: true 
  end
```

Reply:
Thank you so much,